mini_pupper_navigation/launch/localization.launch.py

```diff
@@ -63,7 +63,7 @@ def generate_launch_description():
         DeclareLaunchArgument(
             'load_state_filename',
             default_value=os.path.join(
-                bringup_dir, 'maps', 'cartographer_map.pbstream'),
```

Comment:
> And the map for localization is mymap.pbstream which will be stored in /home directory if the user follows the new README.

There is no `mymap.pbstream` file in the `mini_pupper_navigation/maps` folder.
It also does not contain the above document changes, so it is not consistent.
Please either put the changes to the README regarding navigation in this pull request, or withdraw the changes here.

Reply:
I agree with you, mymap. pbstream file should not be put in the mini_pupper_navigation/maps folder. And I'll change it in README, users will save maps named cartographer_map.pbstream, which will replace official cartographer files in the maps folder, thanks.